aws/v9.3.4/README.md

```diff
@@ -0,0 +1,63 @@
+# :zap: Giant Swarm Release v9.3.4 for AWS :zap:
+
+This release makes security fixes for 2 CVEs:
+- IPv6 rogue router advertisement vulnerability [CVE-2020-13597](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-13597)
+- Intel Microcode vulnerabilities [CVE-2020-0543](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-0543)
```

Comment:
```suggestion
```

@stone-z says this is not there. Is this the flatcar one?

Reply:
Correct, CVE-2020-0543 was the actual vulnerability, the mitigation is from Flatcar